docs/external_rules.rst

```diff
@@ -20,15 +19,15 @@ Refer to :ref:`checkers` for more details.
 This is an example of a file with custom checker that asserts that no test have "Dummy" in the name::
 
     from robocop.checkers import VisitorChecker
-    from robocop.messages import MessageSeverity
+    from robocop.rules import RuleSeverity
```

Comment:
We somehow missed it during our refactor (before "rules" and "RuleSeverity" were called "messages" and "MessageSeverity"). Good job debugging that!